bitnami/common/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts
 type: library
-version: 2.6.0
+version: 2.6.1
```

Comment:
I would bump the minor version as it is adding a feature, not fixing a bug

Reply:
+1

Reply:
done at https://github.com/bitnami/charts/pull/18154/commits/1c4c6b1229fc4c014c7af4eaf3a75bed758d2682